app.rb

```diff
@@ -45,7 +51,36 @@
 
 # Visualization: Project vs Hours Worked
 get '/proj_vs_hours' do
-  erb :proj_vs_hours
+  time_for_each_proj = {}
+  times = ts.get_times
+  projects = ts.get_projects
+
+  times.each do |time|
+    slug = time['project'][0]
+    name = find_project(projects, slug)
+    duration = time['duration']
+
+    # Convert all durations to seconds
+    unless duration.is_a? Integer
```

Comment:
rimesync return all durations in seconds only, but when I do `gt` on `climesync` I get all times in `"HhMm"` format as a result I need to check if its in seconds or not and if its not I convert it.

So I need not place a check when I am using timesync-node as server but if I play around with the times returned from the production server this checks is necessary. I think there is some conflict in the way time is stored in production server?